lib/wpc_proto/internal_modules/proto_config.c

```diff
@@ -276,7 +276,253 @@ bool Proto_config_init(void)
 app_proto_res_e Proto_config_handle_set_config(wp_SetConfigReq *req,
                                                wp_SetConfigResp *resp)
 {
-    return APP_RES_PROTO_NOT_IMPLEMENTED;
+    app_res_e res;
+    app_res_e global_res = APP_RES_OK;
+    bool config_has_changed = false;
+
+    // TODO: Add some sanity checks
+
+    // check any config change and apply them
+    wp_SinkNewConfig * cfg = &req->config;
+
+    //cfg->sink_id[16]; not used
```

Comment:
Should be checked against the only SINK_ID we have (from common).
If not the same, INVALID_SINK_ID should be returned

Reply:
ok

Reply:
done